tests/Supportive/OutputFormatter/TableOutputFormatterTest.php

```diff
@@ -85,7 +85,7 @@ public static function basicDataProvider(): iterable
   Reason      LayerA                            
```

Comment:
The information you are looking for is provided in the header for the section as you can see here.